### ISSUE TITLE
Version issue and replication

### ISSUE THREAD

**Issue by sumegha1024** [xiangning-chen/DrNAS#8]
Hello! 

I am trying to replicate the results of DrNAS for NASbench201 space in Table 4 of your paper. I want to generate the test accuracy for CIFAR10 (94.36% in table 4 of the paper). 

1) Are you using progressive training (train_search_progressive.py) to get those results in Table 4? Additionally you have provided instructions for the evaluation phase in the DARTS space. But what script do you use for the eval phase in NASbench201?
2) Could you share which version of pytorch, cudatoolkit, torchvision, tensorboard etc you used? It would be great if you have a screenshot for the environment you used to get the results.

Thanks a lot for your help!

**Comment by xiangning-chen** [comment]
Hi,

Thanks for your questions.

1. Since memory is not a bottleneck for NAS-Bench-201, so we directly used train_search.py. We don't need to train the architecture in NAS-Bench-201 as it can directly output the result when we query in this database.
2. About the version problem, you can refer to this issues: https://github.com/xiangning-chen/DrNAS/issues/7. I used PyTorch 1.3.0.

**Comment by sumegha1024** [comment]
Thank you so much for your response, this was very helpful! I will try running train_search.py directly.
